src/views/Login.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <b-row>
+    <b-col>
+      <passthrough-form
+        endpoint="/accounts/login/"
+        successRedirectViewName="home"
+      ></passthrough-form>
+      <br/>
+      <!-- TODO: Translate this -->
+      <p>Forgot your password?<router-link :to="{name: 'passwordReset'}"> Reset it</router-link>.</p>
+      <!-- TODO: Translate this -->
```

Comment:
Did you mean to leave these TODOs?

Reply:
Yeah, I want to research a js library to use to do any translation, since we don't have the django internationalization system to do that anymore. I figured that would be a separate PR.